crates/matrix-sdk-ui/src/timeline/controller/observable_items.rs

```diff
@@ -156,40 +156,60 @@ impl Deref for ObservableItemsEntry<'_> {
     }
 }
 
+/// A transaction that allows making multiple updates to an `ObservableItems` as
+/// an atomic unit.
+///
+/// For updates from the transaction to have affect, it has to be finalized with
+/// [`Self::commit`]. If the transaction is dropped without that method being
+/// called, the updates will be discarded.
 #[derive(Debug)]
 pub struct ObservableItemsTransaction<'observable_items> {
     items: ObservableVectorTransaction<'observable_items, Arc<TimelineItem>>,
     all_remote_events: &'observable_items mut AllRemoteEvents,
 }
 
 impl<'observable_items> ObservableItemsTransaction<'observable_items> {
+    /// Get a referene to the timeline index at position `timeline_item_index`.
```

Comment:
I think you meant [`TimelineItem`] instead of timeline index?

Reply:
Indeed. Fixed, thanks!